recipes/graylog-sidecar-repository/recipe.rb

```diff
@@ -0,0 +1,78 @@
+require_relative '../tools'
+
+class GraylogRepository < FPM::Cookery::Recipe
+  include Tools
+
+  name       "graylog-sidecar-repository"
+  VERSION = data.version_major
+
+  description "Package to install Graylog Sidecar GPG key and repository"
+
+  version    '1'
+  revision   1
+  source     '', :with => :noop
+  arch       'all'
+  homepage   data.homepage
+  maintainer data.maintainer
+  vendor     data.vendor
+  license    data.license
+
+  replacements = %w(0.92 0.91 0.90).map {|v|
+    "graylog2-#{v}-repository-#{os}#{osrel}"
+  }.concat(%w(1.0 1.1 1.2 1.3).map {|v|
+    "graylog-#{v}-repository-#{os}#{osrel}"
+  }).concat(%w(2.0 2.1 2.2 2.3 2.4 2.5 3.0 3.1).map {|v|
+    "graylog-#{v}-repository"
+  })
+
+  replaces replacements
+  conflicts replacements
```

Comment:
We don't really need these lines. They have been added to the `graylog-repository` to simplify updating from previous packages. Since we only start this package now, we don't need it.

Reply:
I was wondering about that. It's now removed.